packages/@aws-cdk/aws-ecr/lib/repository.ts

```diff
@@ -512,9 +512,9 @@ export class Repository extends RepositoryBase {
   /**
    * Render the life cycle policy object
    */
-  private renderLifecyclePolicy(): CfnRepository.LifecyclePolicyProperty | undefined {
+  private renderLifecyclePolicy(): { LifecyclePolicyText?: string, RegistryId?: string } | undefined {
     const stack = Stack.of(this);
-    let lifecyclePolicyText: any;
+    let lifecyclePolicyText: string | undefined;
```

Comment:
This is consecutive to the dropping of the modeled `LifeCyclePolicy` property from the CFN Spec. This is because there is no *named* definition in the schema (instead, the property has an inline schema with the correct properties).